README.md

```diff
@@ -93,146 +66,95 @@ After you're finished please remove all the comments and instructions!
   </ul>
 </details>
 
-<!-- Features -->
+<br>
 
 ### Key Features <a name="key-features"></a>
 
 > Describe between 1-3 key features of the application.
 
-- **[key_feature_1]**
-- **[key_feature_2]**
-- **[key_feature_3]**
+- **key_feature_1**
+- **key_feature_2**
+- **key_feature_3**
```

Comment:
[OPTIONAL] - Kindly consider adding  1-3 `key features` of the application to help anyone visiting our repo to have an idea of what the project is about .. 🤗🛠 💪